### DISCUSSION TITLE
Variants Not Working Independently of Each Other

### DISCUSSION THREAD

**Discussion by emirhantetik** [google/model-viewer#4517] [Q&A]
Hello everyone,
When I send the object I created the 3d model to the model viewer editor, I create variants for each part through the editor. However, the variants I have created do not work independently of each other. When I make material selections respectively, there is no problem, but when I want to go back and make a change, the model returns to defaults. I would be very grateful if you can help me how to fix this.

**Comment by elalish** [comment]
I'm not quite following, but that might be a bug. Can you give us an example model and precise steps that repro the issue, along with what the expected behavior would be?
